widgets/GCSMove-action.json

```diff
@@ -128,6 +128,16 @@
           }
         }
       ]
+    },
+    {
+      "label" : "Advanced",
+      "properties" : [
+        {
+          "name": "readTimeout",
+          "widget-type": "number",
+          "label" : "Read Timeout"
```

Comment:
Can also set default minimum and maximum in the widget. See sample size widget referenced in last comment.

Reply:
Set the minimum and default values